inc/vcf/error.hpp

```diff
@@ -1,4 +1,77 @@
 /**
+ ODB License Exception for vcf-validator (Validator for the Variant Call
```

Comment:
This license exception text is huge! It could be moved to the end of the LICENSE file or placed in its own file, but better not to mix it with the code.